runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -359,7 +360,16 @@ createCrossLanguageValidatesRunnerTask(
     "--numprocesses=8",
     "--timeout=4500",
     "--log-cli-level=INFO",
-  ]
+  ],
+  goScriptOptions: [
+    "--runner dataflow",
+    "--project ${dataflowProject}",
+    "--dataflow_project ${dataflowProject}",
+    "--region ${dataflowRegion}",
+    "--tests \"./test/integration/xlang\"",
+    "--dataflow_worker_jar ${project(":runners:google-cloud-dataflow-java:worker").shadowJar.archivePath}",
```

Comment:
Do we need this? I think Dataflow worker jar is already included in Java SDK harness container.

Reply:
Ah, I confused with the sdk harness jar so worker so dataflow worker jar is not included in Java SDK harness by default. Still I wonder why we need Dataflow worker jar. Aren't XLang pipelines only supported on UW?

Reply:
You might be right... I'll have to test this out.

Reply:
Looks like it still works without the Dataflow worker jar. I'll keep that removed.